fuel-vm/src/interpreter/memory.rs

```diff
@@ -451,7 +451,7 @@ fn try_mem_write(
     let ax = addr.checked_add(data.len()).ok_or(PanicReason::ArithmeticOverflow)?;
```

Comment:
I guess you can remove this line, and the same for another function

Reply:
I don't think it can be due to the comment below